e2e/config.go

```diff
@@ -23,11 +32,10 @@ var LvmSubscriptionChannel string
 // DiskInstall indicates whether disks are needed to be installed.
 var DiskInstall bool
 
-// DeployManager is the suite global DeployManager
-var DeployManagerObj *deploymanager.DeployManager
-
 // SuiteFailed indicates whether any test in the current suite has failed
 var SuiteFailed = false
+var scheme = runtime.NewScheme()
```

Comment:
instead of having `var` declared in each line, it can be grouped together.  For example:
` var (
)`

Reply:
Grouped all the vars and const.